plugins/CoreAdminHome/Commands/PurgeOldArchiveData.php

```diff
@@ -58,6 +58,8 @@ protected function configure()
         $this->addOption('exclude-invalidated', null, InputOption::VALUE_NONE, "Do not purge invalidated archive data.");
         $this->addOption('exclude-ranges', null, InputOption::VALUE_NONE, "Do not purge custom ranges.");
         $this->addOption('skip-optimize-tables', null, InputOption::VALUE_NONE, "Do not run OPTIMIZE TABLES query on affected archive tables.");
+        $this->addOption('include-year-archives', null, InputOption::VALUE_NONE, "If supplied, the command will purge archive tables that contain year archives for every supplied date.");
```

Comment:
I'm thinking that this should be the default, ie. always purge year archives for supplied dates. Is there a reason not to do this all the time?